content/blog/embrace-kubernetes-part1/index.md

```diff
@@ -0,0 +1,96 @@
+---
+title: "It’s Time to Embrace Kubernetes! Really? (Part 1)"
```

Comment:
Maybe nitpicking, but "embrace" seems like the wrong word here. Adopt? Deploy? Standardize on?

Reply:
I think it isn't a common word choice for a tech article, but I like the way it implies that Kubernetes is a commitment.